manifests/apache/fragment.pp

```diff
@@ -0,0 +1,20 @@
+# provides the ability to specify fragments for the ssl 
+#   virtual host defined for a Pulp server
+#
+#  === Parameters:
+#
+#  $ssl_content:: content of the ssl virtual host fragment
+define pulp::apache::fragment(
+  $ssl_content = undef,
+  $order       = 15,
+) {
+
+  if $ssl_content {
```

Comment:
My brain pre-optimized to allow HTTP  fragments in the future, I can just remove this check for now :)